packages/drivers/odsp-driver/src/index.ts

```diff
@@ -49,4 +49,13 @@ export {
 	storeLocatorInOdspUrl,
 } from "./odspFluidFileLink";
 
-export { parseCompactSnapshotResponse } from "./compactSnapshotParser";
+export {
```

Comment:
I would recommend adding notes to the PR description about why these new exports were added. Namely that they are transitively referenced by existing exports, and therefore must be exported for API completeness.